src/index.html

```diff
@@ -3,7 +3,12 @@
     <head>
         <meta charset="utf-8">
         <meta data-lisk-hub-version="<%= htmlWebpackPlugin.options.VERSION %>">
-        <meta name="viewport" content="width=device-width,user-scalable=no">
+        <meta name="viewport" content="width=device-width">
+        <meta name="keywords" content="Lisk, Lisk Hub, wallet">
+        <meta property="og:title" content="Lisk Hub">
+        <meta property="og:description" name="description" content="Lisk Hub is an all-in-one solution to manage your Lisk ID, access and send LSK tokens, as well as vote for delegates.">
+        <meta property="og:image" content="https://lisk.io/hub/assets/images/og-logo.png">
+        <meta property="og:url" content="https://lisk.io/hub">
```

Comment:
The URL should not be `https://lisk.io/hub` when Hub is deployed to `https://testnet.lisk.io/hub`. Not sure what the solution should be. Remove`property="og:url"` altogether?

Reply:
I could make it dynamic with javascript